Docs: Templating: Replace __value.raw with __data.fields (#76408)
grafana/grafana: docs/sources/panels-visualizations/configure-data-links/index.md

- `__value.text` - text representation of a value
- `__value.calc` - calculation name if the value is result of calculation

## Data variables

> Data-specific variables are available under `__data` namespace:
> 
> - `__data.fields[i]` - value of data field, `i` is an index starting from 0

Reviewer: ```suggestion
To access values and labels from other fields use 

- `${__data.fields[i]}` - value of field i (on the same row)
- `${__data.fields["NameOfField"]}` - value of field using name instead of index
- `${__data.fields["NameOfField"]}` - value of field using name instead of index
- `${__data.fields[1].labels.cluster}` - Access labels of another field 
```

- `__value.text` - text representation of a value
- `__value.calc` - calculation name if the value is result of calculation

## Data variables

> To access values and labels from other fields use

Reviewer 2: ```suggestion
To access values and labels from other fields use:
```

> - `${__data.fields[i]}` - value of field i (on the same row)

Reviewer 2: I'm not sure what the "i" is referring to here

```suggestion
- `${__data.fields[i]}` - value of field `i` (on the same row)
```

- `${__data.fields["NameOfField"]}` - value of field using name instead of index
- `${__data.fields["NameOfField"]}` - value of field using name instead of index

> - `${__data.fields[1].labels.cluster}` - Access labels of another field

Reviewer 2: ```suggestion
- `${__data.fields[1].labels.cluster}` - access labels of another field
```
